clip.py

```diff
@@ -190,13 +197,16 @@ def saveTo(self, file):
                                     'bpm': self.bpm,
                                     'beat_per_bar': self.beat_per_bar,
                                     'width': self.width,
-                                    'height': self.height}
+                                    'height': self.height,
+                                    'outputs': ",".join(self.outputs)}
             for clip in self.clips:
                 clip_file = {'name': clip.name,
                              'volume': str(clip.volume),
                              'frame_offset': str(clip.frame_offset),
                              'beat_offset': str(clip.beat_offset),
                              'beat_diviser': str(clip.beat_diviser),
+                             'route_out': str(clip.route_out),
```

Comment:
`route_out` rename in `output` should have string value like MAIN, BASS, REAR or rear